README.md

```diff
@@ -12,9 +12,9 @@ See [action.yml](action.yml)
 ```yaml
 steps:
   - name: Check out repo
-    uses: actions/checkout@v2
+    uses: actions/checkout@v3
```

Comment:
`checkout@v3` will lock in us for Node 16 😄

see [this](https://github.com/actions/checkout/compare/v2.4.2...v3.0.2#diff-05c68f898abe14ac98ae36006aed55a2d59f699b8449623ff4eeeb30bf3bbaf2R27)

Reply:
modify to v2